### DISCUSSION TITLE
GitHub Pages failed to build your site

### DISCUSSION THREAD

**Discussion by flywire** [mkdocs/mkdocs#2635] [Q&A] [answered]
After a commit the following [error](https://github.com/flywire/lo-p/runs/3986164261) occurs with three files:

> GitHub Pages failed to build your site.
The variable {{{" + iA + "x} on line 581 in 07-Non-text_Content.md was not properly closed with }}.

I understand this is generated by jekyll interpreting the characters between the curly braces. I formatted the text in a code block but the error still occurred.

The site uses https://squidfunk.github.io/mkdocs-material/ which generates a `.nojekyll` file in the root of the `gh-pages` branch.    (See [Bypassing Jekyll on GitHub Pages](https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/) by creating a file named .nojekyll in the root of your pages repo.) I added a `.nojekyll` file in the root of `main` but the error still occurred.

Any suggestions how to add the required [markdown] files without generating an error in GitHub?

**Comment by oprypin** [comment, marked as answer]
https://github.com/flywire/lo-p/tree/deab17b9d00e422960465b6f131cc0c789751eb7
https://github.com/flywire/lo-p/tree/7a788d66e2693743151c8f4e4ea7cccadf5e5927/docs

The error is complaining about the file "07-Non-text_Content.md". There is no such file on the "gh-pages" branch, it's only on the "main" branch. So it seems you have pointed GitHub to build your "main" branch, but it should build your "gh-pages" branch. If you point it to "main", MkDocs is not involved in any way so the issue would be invalid here.
